packages/manager/apps/web/client/app/dns-zone/new/dns-zone-new.controller.js

```diff
@@ -35,6 +35,7 @@ export default class newDnsZoneCtrl {
 
     this.WORKFLOW_OPTIONS = {
       ...WORKFLOW_OPTIONS,
+      getPlanCode: () => WORKFLOW_OPTIONS.planCode,
```

Comment:
I'm having trouble understanding here, `planCode` is removed from `WORKFLOW_OPTIONS` but is used here

Reply:
Goodness, thankfully you saw this